### DISCUSSION TITLE
Updating from v18 to v20

### DISCUSSION THREAD

**Discussion by Orgbrat** [POPWorldMedia/POPForums#356] [General]
I need to update a .NET 6.0  Web App that has PopForums.Mvc 18.0.0 and PopForums.Sql 18.0.2 within it to .Net 8.0 which I see that PopForums.Mvc v20 has already had done. So can I assume that would be as simple as updating the PopForums.Mvc nuget package. What needs to be updated for the PopForums.Sql  other than it's nuget package to v20. What update sql scripts needs to be ran on the physical MSSQL database? Are there any other changes that needs to be done?

**Comment by jeffputz** [comment]
Upgrade instructions are here: https://popworldmedia.github.io/POPForums/starthere.html

**Comment by Orgbrat** [comment]
So after updating from v18 to v20 so far the only issue was in the Startup.cs within the app.UseEndpoints section as below.  The "AddPopForumsEndpoints" call is invalid and the "forums" parameter in the second endpoints.MapAreaControllerRoute thinks it is invalid.
How or where have those calls changed?

`
app.UseEndpoints(endpoints =>
            {
                // POP Forums routes
                endpoints.AddPopForumsEndpoints(app);

                // need this if you have lots of routing and/or areas
                endpoints.MapAreaControllerRoute(
                    "forumroutes", "forums",
                    "Forums/{controller=Home}/{action=Index}/{id?}");

                // app routes
                endpoints.MapControllerRoute(
                    "areaRoute",
                    "{area:exists}/{controller=Home}/{action=Index}/{id?}");
                endpoints.MapControllerRoute(
                    "default",
                    "{controller=Home}/{action=Index}/{id?}");
                endpoints.MapRazorPages();
            });
`

**Comment by jeffputz** [comment]
Look at the sample project: https://github.com/POPWorldMedia/POPForums.Sample
